pennylane_lightning/_version.py

```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0rc0"
+__version__ = "0.30.0-rc0"
```

Comment:
```suggestion
__version__ = "0.30.0-rc1"
```

Reply:
We need to update this to rc1 now that the changes are into master.

Reply:
Sure makes sense. I will commit this suggestion.